### ISSUE TITLE
v0.3.1 not available for python 3.8 on pypi.org

### ISSUE THREAD

**Issue by mcsage** [alan-turing-institute/sktime#223]
Would it be possible to build the 0.3.1 version for python 3.8 as well and make it available on pypi.org?

```
➜  ~ docker run -ti python:3.8 /bin/bash
root@9c5eb8a054cd:/# pip install sktime==0.3.1
ERROR: Could not find a version that satisfies the requirement sktime==0.3.1 (from versions: 0.1.dev0, 0.1.0, 0.2.0, 0.3.0)
ERROR: No matching distribution found for sktime==0.3.1
```

**Comment by mloning** [comment]
Yes, but probably not before our next release some time in the next couple of months. Any specific reason why you need 3.8?

**Comment by mcsage** [comment]
We are developing with python 3.8 and hitting the numpy dependency issue with sktime 0.3.0 which seems to be solved in v0.3.1
````
ModuleNotFoundError: No module named 'numpy'. Please install numpy first using `pip install numpy`.
```
I know I could install numpy first but this wouldn't be a clean solution for our build toolchain.

**Comment by mloning** [comment]
With release v0.4.0 we now have Python 3.8 wheels ... finally 🎉
